caab-api/open-api-specification.yml

```diff
@@ -315,10 +350,74 @@ paths:
           description: 'Not found'
         '500':
           description: 'Internal server error'
-  /applications/{id}/linked-cases:
+  /applications/{id}/cost-structure:
     get:
       tags:
         - applications
+      summary: 'Get an application''s prior authorities'
+      operationId: 'getApplicationCostStructure'
+      parameters:
+        - name: 'id'
+          in: 'path'
+          required: true
+          schema:
+            type: 'integer'
+            format: 'int64'
+            example: '1234567890'
+      responses:
+        '200':
+          description: 'Successful operation'
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/costStructure"
+        '400':
+          description: 'Bad request'
+        '401':
+          description: 'Unauthorized'
+        '404':
+          description: 'Not found'
+        '500':
+          description: 'Internal server error'
+    put:
+      tags:
+        - applications
+      summary: 'Update an applications cost structure'
+      operationId: 'updateApplicationCostStructure'
+      parameters:
+        - name: 'id'
+          in: 'path'
+          required: true
+          schema:
+            type: 'integer'
+            format: 'int64'
+        - name: 'Caab-User-Login-Id'
+          in: header
+          required: true
+          schema:
+            type: 'string'
+          example: 'SOMEUSER@COMPANY.CO.UK'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/costStructure"
+      responses:
+        '200':
+          description: 'Case updated successfully'
+        '400':
+          description: 'Bad request'
+        '401':
+          description: 'Unauthorized'
+        '404':
+          description: 'Not found'
+        '500':
+          description: 'Internal server error'
+  /applications/{id}/linked-cases:
+    get:
+      tags:
+        - linked cases
```

Comment:
this is still application scoped. Should it not be tagged as application still?

Reply:
amended